index.md

```diff
@@ -101,37 +123,11 @@ because your past self doesn't answer email.
 
-> Text is an area of consistent agony for me,
-> even when collaborating with people who have bought in to plain text,
-> version control, etc.
-> It is exacerbated when I have a collaborator who has never even heard of line endings or encoding.
->
-> Those plain text files we all love?
-> If you've got people editing or even opening/closing them on different platforms,
-> possibly with locale set to something other than US English,
-> things can get weird.
-> Line endings cycle through \n, \r\n, and \r and "smart quotes" drive you crazy.
->
-> Then you get large and uninformative diffs or the dreaded �.
-
-Regarding metadata, [Elizabeth Wickes][wickes-comment-metadata] writes:
+We propose delimited plain text as an appealing "lowest common denominator" data form, that offers high usability across time, people, operating systems, and analytical environments. We must admit that plain text is, however, no panacea. In particular, when collaborating with others, be aware of and, when possible, standardize on file encoding, line endings, and the elimination of prose-oriented features, such as "smart quotes". Differences between collaborators in these pesky details can cause the "diffs" between file versions to be unnecessarily large and, therefore, to be substantially less informative.
 
-> ...two types of metadata that often get conflated in data set discussions:
-> metadata about the data set as a whole
-> and metadata about the content within the data set.
-> Most metadata schemas...are for the former use.
-> They are to describe the data set as a unit,
-> e.g. author, funder, relevant papers, etc.
->
-> Formally structured metadata is often a valueless effort if the data set will be stored independently
-> and not somewhere in a formal repository...
-> Beautifully filled out metadata files are for ingestion into a repository and/or directory.
-> If the audience is humans, write it for humans.
-> If the audience includes metadata harvesters,
-> fill out the formal metadata and do a README for the humans.
+[Elizabeth Wickes][wickes-comment-metadata] provides a useful classification for metadata, with implications for it how it should be represented. She notes it is easy to conflate metadata about the dataset as a whole with metadata about the content, e.g., individual columns. Most metadata schemas are aimed at the former, i.e. they detail the author, funder, related publications, etc. When considering how to store metadata, consider the intended audience. Is it humans? Write a README. Is it machines, such as metadata harvesters and formal repostiories? Create an impeccably formated metadata file.
```

Comment:
Readers of this doc are unlikely to know the difference between peccably and impeccably formatted metadata without an example.